src/uTensor/ops/QuantizeOps.hpp

```diff
@@ -0,0 +1,44 @@
+#ifndef UTENSOR_QUANTIZE_OPS_H
+#define UTENSOR_QUANTIZE_OPS_H
+#include <algorithm>
+#include <limits>
+
+#include "operatorBase.hpp"
+
+namespace uTensor {
+namespace TFLM {
+// https://github.com/tensorflow/tensorflow/blob/fb4ec5cbde3973050e7350f0aca7f07ab7757bac/tensorflow/lite/kernels/internal/reference/dequantize.h#L30-L44
+template <typename OutputT, typename InputT>
+void dequantize_kernel(Tensor& b, const Tensor& a) {
+  
+  // More extensive dimensional checks can be included here
+  const int flat_size = b->get_shape().get_linear_size();
+  const QuantizationParams& quantParam = a->get_quantization_params();
+  int32_t zero_point = quantParam.get_zeroP_for_channel(0);
+  float scale = quantParam.get_scale_for_channel(0);
+
+  for (int i = 0; i < flat_size; i++) {
+    const int32_t val = static_cast<InputT>(a(i));
+    OutputT result = static_cast<OutputT>(scale * static_cast<float>((val - zero_point)));
```

Comment:
Is this float cast required? I thought the TFLM version could avoid float computations

Reply:
`scale` has to be a float, so the casting is going to happen one way or another. The good news is, this is op is pretty optional, often only happen once per graph if it ever shows now.

Reply:
Do we need the `TFLM` namespace here?
Are we going to support different kind of quant/dequant for different frameworks this way?

Reply:
Doesn't seem like a bad idea. What uTensor provide are the infrastructure and interfaces. We should anticipate that others might want to add their own sets of ops in the future.

Reply:
I think it's fine since it signifies that this op is bit compatible with TFLu's implementation. I did something similar with the legacy code.